docker-compose-single.yml

```diff
@@ -1,8 +1,6 @@
 version: "3.3"
 services:
   iroha:
-    build:
-      context: .
     image: iroha:debug
```

Comment:
same as above

Reply:
https://github.com/hyperledger/iroha/pull/1579#discussion_r743792278
So if there were something to be removed, it would be line 6 as this name does not used anyway in `-single.yml`

Reply:
`image` gives name to the built image so I would leave it.
`build` is required because without it docker will not try to build an image if pull fails

Reply:
Yeah, and in my understanding it won't pull when `build` is specified